ci/Jenkinsfile.uitests

```diff
@@ -27,6 +27,14 @@ pipeline {
       daysToKeepStr: '30',
       artifactNumToKeepStr: '3',
     ))
+    /* Throttle number of concurrent builds. */
+    throttleJobProperty(
+      throttleEnabled: true,
+      throttleOption: 'category',
+      categories: ['status-desktop-e2e-tests'],
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 1
+    )
```

Comment:
Are you sure that it's a good idea for this PR? For example, I'm can't give a good review for CI and it can be merged without `good review`